src/rust/src/hardsubx/utility.rs

```diff
@@ -0,0 +1,96 @@
+// use crate::{
+//     bindings::*
+// };
+
+#[cfg(feature = "hardsubx_ocr")]
```

Comment:
You have already added `#[cfg(feature = "hardsubx_ocr")` on `mod hardsubx`. Is it needed to be added to all functions in this file?